test/election.test.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from 'chai';
```

Comment:
tests do not have any cases that simulate network partitions yet -- I discovered some interesting edge cases when writing tests against this algorithm that introduce connection interrupts similar to the lease tests in this [lib's core](https://github.com/mixer/etcd3/blob/master/test/lease.test.ts#L67)